### PR TITLE
Feature/bindings paper components

### PR DESCRIPTION
**Pull request by medson10**

### CODE REVIEW

**Comment by fakenickels** [comment]
break the components to each one have its own file

**Comment by fakenickels** [comment]
we won't need the group module PaperComponents because we can use the namespace feature of bsconfig

**Comment by fakenickels** [comment]
https://bucklescript.github.io/docs/en/build-configuration.html#name-namespace

File: re/Button.re [hunk 1]
```diff
@@ -0,0 +1,40 @@
+module Button = {
```
Comment (fakenickels) [inline re/Button.re]: you can put everything out of Button, flat in the file, Button.re is already the module Button

File: re/Button.re [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          "icon": from_opt(icon),
+          "color": from_opt(color),
+          "style": from_opt(style),
+          "theme": from_opt(theme), 
+          "onPress": from_opt(onPress)
+        }
+      ),
+    children
+  );
+};
```
Comment (fakenickels) [inline re/Button.re]: missing EOL break

**Review by fakenickels — APPROVED** [review]

**Comment by guilhermedecampo** [comment]
Top!

**Pull request merged**